Source: Jakob Nixdorf <flocke@shadowice.org>
Upstream: No
Reason: Prevent Spyder from installing the Windows-specific post-install script

diff -Nur spyder-2.3.8.orig/setup.py spyder-2.3.8/setup.py
--- spyder-2.3.8.orig/setup.py	2016-02-04 06:28:54.927003383 +0100
+++ spyder-2.3.8/setup.py	2016-02-04 06:29:29.056748167 +0100
@@ -206,7 +206,7 @@
 
 # NOTE: the '[...]_win_post_install.py' script is installed even on non-Windows
 # platforms due to a bug in pip installation process (see Issue 1158)
-SCRIPTS = ['%s_win_post_install.py' % NAME]
+SCRIPTS = []
 if PY3 and sys.platform.startswith('linux'):
     SCRIPTS.append('spyder3')
 else:
